eng/pipelines/templates/stages/archetype-sdk-client.yml

```
@@ -4,17 +4,34 @@ parameters:
 - name: Artifacts
   type: object
   default: []
+# Test Parameters
 - name: TestTimeoutInMinutes
   type: number
   default: 60
-- name: oneESTemplateTag
-  type: string
-  default: release
+- name: MatrixConfigs
+  type: object
+  default:
+  - Name: rust_ci_test_base
+    Path: eng/pipelines/templates/stages/platform-matrix.json
+    Selection: sparse
+    NonSparseParameters: RustToolchainName
+    GenerateVMJobs: true
```

The way the matrix is set up now, you could specify it with `all` instead. Though perhaps you have it with `NonSparseParameters` to handle when more matrix parameters are added?

```suggestion
  default:
  - Name: rust_ci_test_base
    Path: eng/pipelines/templates/stages/platform-matrix.json
    Selection: all
    GenerateVMJobs: true
```

---

I wasn't sure how selection: all would interact with sparse coming from the PR matrix generation